<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkyImgDialog</class>
 <widget class="QWidget" name="SkyImgDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sky Image</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="Panel1" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="Label1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Image Size W/ H (pixels)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="SkySize1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Label2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="SkySize2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SkyFlip">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Flip</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="Panel2" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="Label6">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Roll</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="SkyCent1">
        <property name="minimum">
         <double>-1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="SkyFov1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Label9">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="Label7">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Plot Center (pixels)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="7">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="6">
          <widget class="QLabel" name="Label17">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>20</string>
           </property>
          </widget>
         </item>
         <item row="1" column="9">
          <widget class="QLabel" name="Label19">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="Label16">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>30</string>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QLabel" name="Label18">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="Label15">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>40</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="Label14">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>50</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="Label13">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>60</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="Label12">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>70</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="Label11">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>80</string>
           </property>
          </widget>
         </item>
         <item row="0" column="9">
          <widget class="QLabel" name="Label20">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>(deg)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QDoubleSpinBox" name="SkyDest1">
           <property name="decimals">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="SkyDest2">
           <property name="decimals">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QDoubleSpinBox" name="SkyDest3">
           <property name="decimals">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QDoubleSpinBox" name="SkyDest4">
           <property name="decimals">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QDoubleSpinBox" name="SkyDest5">
           <property name="decimals">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <widget class="QDoubleSpinBox" name="SkyDest6">
           <property name="decimals">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="3" column="6">
          <widget class="QDoubleSpinBox" name="SkyDest7">
           <property name="decimals">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="3" column="7">
          <widget class="QDoubleSpinBox" name="SkyDest8">
           <property name="decimals">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="3" column="9">
          <widget class="QDoubleSpinBox" name="SkyDest9">
           <property name="decimals">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="Label5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="SkyCent2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="minimum">
         <double>-1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="Label3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Pitch</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="SkyFov2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="Label4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Yaw</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QDoubleSpinBox" name="SkyScale">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="minimum">
         <double>-1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QDoubleSpinBox" name="SkyFov3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="minimum">
         <double>-180.000000000000000</double>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="6">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="SkyDestCorr">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Pixels from Center - Elevation</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="SkyElMask">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Elevation Mask</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="Label8">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Resample</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="SkyRes">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <item>
            <property name="text">
             <string>NN</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BL</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QCheckBox" name="SkyBinarize">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>Binarize</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QDoubleSpinBox" name="SkyBinThres1">
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QDoubleSpinBox" name="SkyBinThres2">
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="4" colspan="2">
       <widget class="QLabel" name="Label10">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Plot Radius (pixels)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="BtnSave">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Save Tag</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnLoad">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Load Tag...</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnGenMask">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Gen Mask</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnUpdate">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Update</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnClose">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Close</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>SkySize1</tabstop>
  <tabstop>SkySize2</tabstop>
  <tabstop>SkyFlip</tabstop>
  <tabstop>SkyCent1</tabstop>
  <tabstop>SkyCent2</tabstop>
  <tabstop>SkyScale</tabstop>
  <tabstop>SkyFov1</tabstop>
  <tabstop>SkyFov2</tabstop>
  <tabstop>SkyFov3</tabstop>
  <tabstop>SkyDest1</tabstop>
  <tabstop>SkyDest2</tabstop>
  <tabstop>SkyDest3</tabstop>
  <tabstop>SkyDest4</tabstop>
  <tabstop>SkyDest5</tabstop>
  <tabstop>SkyDest6</tabstop>
  <tabstop>SkyDest7</tabstop>
  <tabstop>SkyDest8</tabstop>
  <tabstop>SkyDest9</tabstop>
  <tabstop>SkyDestCorr</tabstop>
  <tabstop>SkyElMask</tabstop>
  <tabstop>SkyRes</tabstop>
  <tabstop>SkyBinarize</tabstop>
  <tabstop>SkyBinThres1</tabstop>
  <tabstop>SkyBinThres2</tabstop>
  <tabstop>BtnSave</tabstop>
  <tabstop>BtnLoad</tabstop>
  <tabstop>BtnGenMask</tabstop>
  <tabstop>BtnUpdate</tabstop>
  <tabstop>BtnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
